go.mod

```diff
@@ -3,16 +3,14 @@ module github.com/conductorone/baton-privx
 go 1.21
 
 require (
-	github.com/SSHcom/privx-sdk-go v1.35.0
-	github.com/conductorone/baton-sdk v0.2.2
+	github.com/SSHcom/privx-sdk-go v1.35.1
+	github.com/conductorone/baton-sdk v0.2.9
```

Comment:
The newer version of baton-sdk has a slighly different DefineConfiguration function. It accepts a whole config, not just schema fields. You'll have to tweak that before it'll build.

Reply:
Here's an example of the necessary change: https://github.com/ConductorOne/baton-ldap/pull/50/files#diff-b80bfea8c51265df2d90c7b2c3eaa095a0e1c4506adb1d9381348771f07e9de4R22